src/main/java/entities/GameAccess.java

```diff
@@ -0,0 +1,90 @@
+package entities;
+
+import java.util.ArrayList;
+
+/**
+ * Defines functions to manipulate a Game.
+ */
+public interface GameAccess {
```

Comment:
Nice, this looks like a nice way to abstract the Game manipulation and reduce the load on the Game class.